distribution/lib/Standard/Base/0.0.0-dev/src/Data/Statistics.enso

```diff
@@ -388,3 +344,78 @@ is_valid v = case v of
     Nothing -> False
     _ : Number -> v.is_nan.not
     _ -> True
+
+## PRIVATE
+type Accumulator
+    new : Integer -> Accumulator
+    new moments=0 = Accumulator.Value 0 Nothing Nothing (Vector.fill moments 0)
+
+    ## PRIVATE
+    Value count:Integer minimum:Any maximum:Any (moments:Vector|Nothing)
+
+    ## PRIVATE
+    comparator_error : Boolean
+    comparator_error self = self.minimum.is_nothing
+
```

Comment:
This is a change from previous behavior where we had a separate flag for this.

I think we are lacking an important test - what happens when we have a Vector of all Nothing/NaN values?? We have a test for empty vector but not this.

```
[Nothing, Nothing, Nothing].compute Statistic.Minimum . should_???
[Nothing, Nothing, Nothing].compute Statistic.Maximum . should_???
[Nothing, Nothing, Nothing].compute Statistic.Count . should_???
[Nothing, Nothing, Nothing].compute Statistic.Skew . should_???

[Number.nan].compute Statistic.Maximum . should_???
[Number.nan].compute Statistic.Minimum . should_???
[Number.nan].compute Statistic.Count . should_???
[Number.nan].compute Statistic.Skew . should_???

[Nothing, Nothing, Nothing].running Statistic.Minimum . should_???
[Nothing, Nothing, Nothing].running Statistic.Maximum . should_???
[Nothing, Nothing, Nothing].running Statistic.Count . should_???
[Nothing, Nothing, Nothing].running Statistic.Skew . should_???
```

I hypothesize this PR is changing the semantics in this case - if I read the code correctly, the Minimum/Maximum cases will now throw `Incomparable_Values` - which is wrong - `Nothing` values are not incomparable - if we had any other value it would not throw anymore. I guess `Empty_Error` (old behaviour) is more appropriate here.

Reply:
We haven't changed the values on this but agree were missing the tests. There is a filter for Nothing and NaN to be excluded from the accumulation (`is_valid`). I've added checks for these doing the current logic.